Upstream-Status: Pending

Poky provides a list of site files in CONFIG_SITE whereas autoconf 
only expects one file. This patch changes autoconf to accept a list of 
them.

RP 1/2/10

Index: autoconf-2.65/lib/autoconf/general.m4
===================================================================
--- autoconf-2.65.orig/lib/autoconf/general.m4	2010-02-01 12:41:32.329073138 +0000
+++ autoconf-2.65/lib/autoconf/general.m4	2010-02-01 12:41:56.769040799 +0000
@@ -1863,17 +1863,10 @@
 m4_define([AC_SITE_LOAD],
 [# Prefer an explicitly selected file to automatically selected ones.
 ac_site_file1=NONE
-ac_site_file2=NONE
 if test -n "$CONFIG_SITE"; then
   ac_site_file1=$CONFIG_SITE
-elif test "x$prefix" != xNONE; then
-  ac_site_file1=$prefix/share/config.site
-  ac_site_file2=$prefix/etc/config.site
-else
-  ac_site_file1=$ac_default_prefix/share/config.site
-  ac_site_file2=$ac_default_prefix/etc/config.site
 fi
-for ac_site_file in "$ac_site_file1" "$ac_site_file2"
+for ac_site_file in $ac_site_file1
 do
   test "x$ac_site_file" = xNONE && continue
   if test /dev/null != "$ac_site_file" && test -r "$ac_site_file"; then
